submissions/LGeorge007/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,71 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/LGeorge007/a-tiny-JS-world
+
+   Web app: https://lgeorge007.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: "dog",
+   name: "Vasiliy",
+   legs: 4,
+   hands: 0,
+   gender: "male",
+   friends: ["Bublik", "Beem"],
+   saying: "Woof woof!",
+};
+
+const cat = {
+   species: "cat",
+   name: "Murkka",
+   legs: 4,
+   hands: 0,
+   gender: "female",
+   friends: ["Barsik", "Murzik", "Felix"],
+   saying: "Meow meow!",
+};
+
+const woman = {
+   species: "human",
+   name: "Larisa",
+   legs: 2,
+   hands: 2,
+   gender: "female",
+   friends: ["Irina", "Vlad", "Marina"],
+   saying: "Hi!",
+};
+
+const catWoman = {
+   species: "superhero human",
+   name: "Cat woman",
+   legs: 2,
+   hands: 2,
+   gender: "female",
+   friends: ["Batman", "all cats in the world"],
+   saying: cat.saying,
+};
+
+const man = {
+   species: "human",
+   name: "Boris",
+   legs: 2,
+   hands: 2,
+   gender: "male",
+   friends: ["Vladimir", "Mariya", "Oksana"],
+   saying: "Hello!",
+};
+
+const message = function(inhabitant) {
+   return `I'm a ${inhabitant.species}. My name is ${inhabitant.name}. My gender is ${inhabitant.gender}. ` + ((inhabitant.hands !== 0)? `I have ${inhabitant.hands} hands and ${inhabitant.legs} legs. ` : `I have ${inhabitant.legs} paws. `) + `Also I have a friends: ${inhabitant.friends.join(", ")}. ${inhabitant.saying}`;
+};
```

Comment:
Let's make this fragment more readable and extensible (imagine we want to add a couple more props)

Check the example below:
```

const someVar = condition ? 'option1' : 'option2';

return [
  `I'm a ${inhabitant.species}`,
  `My name is ${inhabitant.name}`,
  inhabitant.hands 
    ? `${inhabitant.hands} hands and ${inhabitant.legs} legs`
    : `${inhabitant.legs} paws`,
  `more text with a ${someVar}`,
].join('. ');
```

Besides, there is quite a repetition of `inhabitant`. How about decomposition?

```
const longerStringLiteral = `${obj.propA} some text ${obj.propB}`;

const { propA, propB } = object;

const shorterStringLiteral = `${propA} some text ${propB}`;
```

Do not copypaste the examples but comprehend those and use to your benefit.

Reply:
Thank you!
Your code is really more convenient and prettier!
I've figured out destruction (ES6) in practice.
I didn't know that you can use conditions directly in the array. I forgot that if (0) === false.
Thanks for your prompt.

Please, review.